Marlin/Configuration_adv.h

```diff
@@ -791,7 +791,7 @@
 
 // The ASCII buffer for serial input
 #define MAX_CMD_SIZE 96
-#define BUFSIZE 4
+#define BUFSIZE 6
```

Comment:
What is the motivation behind this?

Reply:
Have read that it helped stabilize connection with hosts. Have been testing in the farm and it seems to work.